experimental/framework/frs-client/src/FrsAudience.ts

```diff
@@ -7,12 +7,13 @@ import { ServiceAudience } from "@fluid-experimental/fluid-framework";
 import { IClient } from "@fluidframework/protocol-definitions";
 import { IFrsAudience, FrsMember } from "./interfaces";
 
-export class FrsAudience extends ServiceAudience<FrsMember> implements IFrsAudience {
-  protected createServiceMember(audienceMember: IClient): FrsMember {
+export class FrsAudience extends ServiceAudience<FrsMember<any>> implements IFrsAudience {
```

Comment:
The `FrsAudience` should preserve passing the T type down. Otherwise, additionalDetails will just get typed as any and not `T`
```suggestion
export class FrsAudience<T = any> extends ServiceAudience<FrsMember<T>> implements IFrsAudience<T> {
```